Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,7 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+const wholeNumberPart = Math.floor(num);
+const decimalPart = +(num - wholeNumberPart).toFixed(2); 
```

Comment:
The method used is correct, but is that the correct number of decimal places?

Reply:
Hello thank you for the review, I did the (exercises/decimal.js ) again. please review it again. I wrote in the terminal and it was executed.